src/pages/movieList.js

```diff
@@ -2,16 +2,25 @@
 
 import {createMovieCard} from '../components/card.js';
 import {renderPageLayout} from '../layouts/createLayout.js';
+import {getTopRatedFilms} from '../services/api.js';
 
 export function createMovieList() {
     const arrayWithCards = [];
 
-    for (let i = 0; i < 20; i++) {
-        arrayWithCards.push(createMovieCard());
-    }
+    getTopRatedFilms().then(result => {
 
-    renderPageLayout(arrayWithCards);
+        for (let i = 0; i < result.length; i++) {
+            const oneFilm = result[i];
+            const posterOfFilm = oneFilm[0];
+            const filmName = oneFilm[1];
+            const filmOverview = oneFilm[2];
 
-    const main = document.querySelector('main');
-    main.className = 'main--movie-list';
+            arrayWithCards.push(createMovieCard(posterOfFilm, filmName, filmOverview));
+        }
+    
+            renderPageLayout(arrayWithCards);
+
+        const main = document.querySelector('main');
```

Comment:
@Mmarvinn better to have opportunity to pass a class name to `renderPageLayout` fn

Reply:
fix